src/Service/Apps/Quap/QuapComputeAnswersService.php

```diff
@@ -86,39 +86,39 @@ public function computeAnswer(string $evaluationFunction, Group $group): int
             case 'split_lead_coach':
                 return $this->splitLeadCoach($group);
             case 'has_leader':
-                return $this->hasLeader($group);
+                return $this->hasAnyRole($group, [Role::CANTONAL_LEADER, Role::REGIONAL_LEADER]);
```

Comment:
@SebastianStorz I think you removed some logic here, there need to be 2 people with the leader role as far as i know